src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -1575,17 +1575,17 @@ fn resolution_failure(
                         _ => None,
                     };
                     // See if this was a module: `[path]` or `[std::io::nope]`
-                    if let Some(module) = last_found_module {
-                        let module_name = collector.cx.tcx.item_name(module);
+                    if let Some(_module) = last_found_module {
                         let note = format!(
-                            "the module `{}` contains no item named `{}`",
-                            module_name, unresolved
+                            "there is no item named `{}` in scope",
+                            unresolved
                         );
                         if let Some(span) = sp {
                             diag.span_label(span, &note);
                         } else {
                             diag.note(&note);
                         }
+                        diag.help(&format!("did you mean to import `{}`?", unresolved));
```

Comment:
I should probably remove this part since it's not super helpful and could be confusing, but I figured that I would post it so you could see what it looks like.